.env.example

```
@@ -1,61 +1,14 @@
+APP_ELTOR_USE_PHOENIXD_EMBEDDED="false"
+APP_ELTOR_LN_IMPLEMENTATION="cln"
+APP_ELTOR_LN_CONFIG="type=cln url=https://YOUR_URL:PORT rune=YOUR_RUNE default=true"
+APP_ELTOR_LN_BOLT12="lno***"
+APP_ELTOR_USER_DIR="$PWD"
+APP_ELTOR_ELTORRC_PATH="$PWD/backend/bin/data"
+
+# Tor Control Passwords (used for authentication to Tor control ports)
+APP_ELTOR_TOR_CONTROL_PASSWORD="password1234_"
+APP_ELTOR_TOR_RELAY_CONTROL_PASSWORD="password1234_"
+
+ACCEPT_INVALID_CERTS=true
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)